examples/scroll-sync.amp.html

```diff
@@ -22,7 +22,10 @@
   </div>
 
   <div>
-    <amp-fx-scroll-sync name='scale'>
+    <amp-fx-scroll-sync name='scale'
```

Comment:
when do reach to the `end-scale`? don't we need an `ending-position` to determine it?